assets/js/modules/analytics/datastore/accounts.test.js

```diff
@@ -50,6 +49,8 @@ describe( 'modules/analytics accounts', () => {
 	let registry;
 	let store;
 
+	const propertiesEndpoint = /^\/google-site-kit\/v1\/modules\/analytics\/data\/accounts-properties-profiles/;
```

Comment:
Copied from `assets/js/modules/analytics-4/datastore/properties.test.js` where all the endpoints are declared as constants. Could be worth doing this for all of them here as I've had to duplicate these regex expressions a lot